src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -73,6 +76,9 @@
                             ResourceKey="SystemColorWindowTextColorBrush" />
             <StaticResource x:Key="SettingContainerMessageForeground"
                             ResourceKey="SystemColorWindowTextColorBrush" />
+
+            <StaticResource x:Key="SettingContainerResetButtonIconForeground"
+                            ResourceKey="SystemAccentColorLight1" />

```

Comment:
I do not know if this is correct in High Contrast - is there a more "contrasty" button content resource we should use?

Reply:
Looks like WinUI doesn't provide a recommendation here. But we got approval from the a11y folks: https://github.com/microsoft/terminal/issues/17902#issuecomment-2358204186

I also like this value because it keeps the accent color in use, so the app still looks and feels personalized to the user.